README.md

```diff
@@ -0,0 +1,29 @@
+# Red Hat–PyCon CZ Coding Challenge
+
+We have prepared 3 programming problems for you to solve so you can win a prize and maybe even a job at Red Hat!
+
+## Rules
+
+To solve the 3 problems you may use *only* the Python 3 programming language and it's standard library, __no external libraries are allowed__. Your script also has to finish within 2 minutes on a reasonably powerful machine.
+
+**Submit** your scripts through our [web interface](https://pyconcz.fedoralovespython.org/) before the **deadline on Friday at 14:00h**.
```

Comment:
I did that on purpose because some Americans have no idea what 14:00 is. However, maybe I'm being too paranoid?